src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContext.ReadWrite.cs

```diff
@@ -160,9 +160,12 @@ internal Task FlushAsync(CancellationToken cancellationToken = default(Cancellat
                 }
             }
             // We want to swallow IO exception and allow app to finish writing
-            catch (Exception ex) when (!(ex is IOException))
+            catch (Exception ex)
             {
-                _bodyOutput.Reader.Complete(ex);
+                if (!(ex is IOException))
```

Comment:
Do we want to log anything when there is an IO exception?

Reply:
@Tratcher should we?

Reply:
There should be one disconnect log once per failed request, but not on every write.

Reply:
And it should be Debug, not an Error.,

Reply:
https://github.com/aspnet/IISIntegration/issues/1065